src/cascadia/PublicTerminalCore/HwndTerminal.cpp

```diff
@@ -769,9 +770,50 @@ HRESULT _stdcall TerminalResize(void* terminal, COORD dimensions)
     publicTerminal->_terminal->ClearSelection();
     publicTerminal->_renderer->TriggerRedrawAll();
 
+    // Resize the text buffer without changing the renderer draw size.
     return publicTerminal->_terminal->UserResize(dimensions);
 }
 
+/// <summary>
+/// Resizes the renderer to fit the provided width and height without changing the text
+/// buffer size.
+/// </summary>
+HRESULT _stdcall ResizeRendererDrawSpace(void* terminal, double width, double height, _Out_ COORD* dimensions)

```

Comment:
I know you didn't start the fire, but it's weird to have SAL declared on the `_Out_` parameter only. I think all four should probably be SAL'd. (Normally we'd do something more modern C++ and discard the SAL entirely, but I realize this is an exported C interface, so I'm not considering that.)